chat.cpp

```diff
@@ -16,6 +16,7 @@
 #include <unistd.h>
 #elif defined (_WIN32)
 #include <signal.h>
+#include <Windows.h>
```

Comment:
import conditionally

Reply:
It is under the preprocessor directive #elif defined (_WIN32).